actions/admin/contact.go

```diff
@@ -210,3 +211,39 @@ func contactsAccept(c *gin.Context, _ *reqctx.AdminContext) {
 
 	c.JSON(http.StatusOK, contract)
 }
+
+// contactsConfirm will perform Confirm action on contacts with the given xpub ids and paymails
+// Perform confirm action on contacts godoc
+// @Summary		Confirm contacts
+// @Description Confirm contacts
+// @Tags		Admin
+// @Produce		json
+// @Param		[]models.ContactConfirmationData body []models.ContactConfirmationData true "Contacts data"
+// @Success		200
+// @Failure		400	"Bad request - Error while getting data from request body"
+// @Failure		404	"Not found - Error, contacts not found"
+// @Failure 	500	"Internal server error - Error, confirming contact failed"
+// @Router		/api/v1/admin/contacts/confirmations [post]
+// @Security	x-auth-xpub
+func contactsConfirm(c *gin.Context, _ *reqctx.AdminContext) {
+	logger := reqctx.Logger(c)
+
+	var reqParams []*models.ContactConfirmationData
```

Comment:
I would rather limit the number of contacts that could be confirmed at once to 2 contacts.
Maybe having object (instead of list) with fields `partyA` and `partyB` would be then more intuitive that you can only confirm 2 counterparties.
Also this should rather confirm the contacts against each other, not confirm any contact. So given party A and party B we want A to have B confirmed and B to have A confirmed, no other contacts.

BUT WHY?
Because confirmation is very important operation that is assuring the you will make a transfers to correct person/address, and unfortunately server is not able to prove that the counterparties really made this confirmation, and what's more, admin is unable to prove to server that they do that, so it's important to limit this operation scope to a bare minimum.

Also this would kinda solve the problem o N queries because N is always 2  😄 


And yes, I know:
![me](https://pbs.twimg.com/media/EeuxzkpXsAA90Ox.jpg)
(🇬🇧 Sorry, AFAIK, there is not English substitute for that )

Reply:
Hmmm... instead of array with a "not-explicit-from-user-perspective" limit.
Maybe: a model:
```goalng
type AdminConfirmContactPair struct {
    A ContactConfirmationData
    B ContactConfirmationData
}
```

Reply:
array swapped to:
```
type AdminConfirmContactPair struct {
	ContactA ContactConfirmationData `json:"contactA"`
	ContactB ContactConfirmationData `json:"contactB"`
}
```